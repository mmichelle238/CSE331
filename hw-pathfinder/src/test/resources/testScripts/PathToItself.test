# Find a path with Dijkstra's algorithm between
# the same node.

CreateGraph simpleGraph

AddNode simpleGraph A

FindPath simpleGraph A A