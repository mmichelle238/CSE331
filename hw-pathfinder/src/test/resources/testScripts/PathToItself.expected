# Find a path with Dijkstra's algorithm between
# the same node.

created graph simpleGraph

added node A to simpleGraph

path from A to A:
total cost: 0.000